js/_actions.js

```diff
@@ -6,6 +6,8 @@ const gitAPI = api.create('git');
 const planAPI = api.create('plan');
 const deployAPI = api.create('deploys');
 
+export const history = null;
```

Comment:
yeah, it's an exported variable that is set by EnvironmentOverview.jsx. I'll add some comments